shared/chat/conversation/messages/wrapper/index.js

```diff
@@ -186,6 +187,10 @@ class _WrapperMessage extends React.Component<Props & Kb.OverlayParentProps, Sta
       />
     )
 
+  _coinFlip = () =>
+    // $ForceType
+    this.props.message.flipGameID && <CoinFlip key="CoinFlip" flipGameID={this.props.message.flipGameID} />
```

Comment:
!!

Reply:
?

Reply:
If this is about the `ForceType`, it is also on a bunch other functions like this. The reason is this `flipGameID` doesn't show up on every incarnation of `Types.Message` so it's just easier to just silence Flow here.